config/routes.rb

```diff
@@ -36,4 +36,15 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "applications#new"
+  post "/applications", to: "applications#create"
+  get "/applications/:id", to: "applications#show"
+  patch "/applications/:id", to: "applications#update"
+  
+  get "/:admin/applications/:id", to: "applications#show"
+  patch "/:admin/applications/:app_id/:pet_id/:app_rej", to: "application_pets#update"
+  
+  get "/:admin/shelters", to: "shelters#index"
+  get "/:admin/shelters/:id", to: "shelters#show"
```

Comment:
I think Juliet talked to you about this during a check in, we would want to use name spacing for all admin functionality. So you would create a directory called admin and then have a file called shelters. 
```ruby
controllers/admin/shelters_controller.rb
controllers/admin/applications_controller.rb
```